common/testData/personNameTest/af.txt

```diff
@@ -53,8 +53,8 @@
 # =====
 
 enum ; field ; title, given, given2, surname, surname2, generation, credentials
-enum ; modifiers ; informal, allCaps, initialCap, initial, monogram, prefix, core
-enum ; options ; none, sorting
+enum ; modifiers ; informal, allCaps, initialCap, initial, retain, monogram, prefix, core, vocative, genitive
+enum ; order ; givenFirst, surnameFirst, sorting
```

Comment:
If you have the ability, it might be better to call these `forceGiven` and `forceSur`, or something along those lines, for clarity and consistency.

Reply:
Good idea. That is a bit more than I want to do right now, but I'll write a ticket to handle in the future.

Reply:
https://unicode-org.atlassian.net/browse/CLDR-17032

Reply:
Thank you!